subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorFragments.kt

```diff
@@ -71,6 +72,7 @@ fun fragmentsForConfiguration(accessor: Accessor.ForConfiguration): Fragments =
     val (functionFlags, deprecationBlock) =
         if (config.hasDeclarationDeprecations()) publicFunctionWithAnnotationsFlags to config.getDeclarationDeprecationBlock()
         else publicFunctionFlags to ""
+    val compileAgainstGradleVersion = GradleVersion.version(System.getProperty("org.gradle.api.version", GradleVersion.current().version)).baseVersion
```

Comment:
I think we should call this "target Gradle version" everywhere.

Reply:
Renamed.